src/app/core/entity/default-datatype/edit-component.ts

```diff
@@ -62,9 +58,6 @@ export abstract class EditComponent<T> implements OnInit, OnChanges {
     if (!this.formFieldConfig?.forTable) {
       this.label = this.formFieldConfig?.label ?? this.propertySchema?.label;
     }
-    if (this.formFieldConfig?.forTable) {
-      this.tooltip = undefined;
-    }
```

Comment:
As far as I understand this now, the description is only not shown in the edit mode of the entity subrecord because it does not use the `FormFielComponent` and therefore the description is not rendered? If we would want to use the component there as well we would still need to remove the description. Maybe this should already happen in the `EntityFormService`?

Reply:
I think this tooltip property was not used here anyway. Tooltip has always been displayed one or two parent levels higher. For table inline edit mode it should be omitted for layout reasons, I guess.

I am moving this logic to the FormFieldComponent, where the tooltip is actually displayed and the full formFieldConfig parsed. It was actually still missing there. Thanks!